plugins/pip.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "pip",
+    "version": "0.0.1",
+    "match": "^python3[0-9]{1,2}Packages.pip",
+    "readme": "This plugin adds a script for automatically creating a virtual environment using `venv` for python3 projects, so you can install packages with pip as normal.\nTo activate the environment, run `source $VENV_DIR/bin/activate` or add it to the init_hook of your devbox.json\nTo change where your virtual environment is created, modify the $VENV_DIR environment variable in your init_hook",
+    "env": {
+        "VENV_DIR": "{{ .Virtenv }}/pip/.venv"
+    },
+    "create_files": {
+        "{{ .DevboxDir }}/venvShellHook.sh": "pip/venvShellHook.sh"
```

Comment:
Do we want this in `devbox.d` (committed) vs `.devbox` (not-committed) ? Since the file is not meant to be modified, I would think `.devbox` (which is `{{ .Virtenv }}`).

Reply:
I was thinking committed for visibility + if someone wants to modify the script, but could definitely go in `{{ .Virtenv }}`

Reply:
I think i like this better in {{ .Virtenv }}. If we get a request we can move it to devbox.d or let someone define their own script.